test/definitions/checks/disk_speed_minimal_test.rb

```diff
@@ -1,38 +0,0 @@
-require 'test_helper'
```

Comment:
@iNecas Tons of thanks for the pointer. **It worked** :smile:
_Test case restored_